lib/charms/loki_k8s/v0/loki.py

```diff
@@ -417,15 +414,13 @@ def alerts(self) -> dict:
 
 
 class LokiConsumer(RelationManagerBase):
-    """
-    Loki Consumer class
-    """
+    """Loki Consumer class."""
 
     _stored = StoredState()
     _ALERT_RULES_PATH: str
 
     def __init__(
-        self, charm: CharmBase, relation_name: str, alert_rules_path="src/loki_alert_rules"
+        self, charm: CharmBase, relation_name: str, alert_rules_path: str = "src/loki_alert_rules"
```

Comment:
consider adding a default for `relation_name`

Reply:
ack